ball.java

```diff
@@ -0,0 +1,12 @@
+public class ball {
+    int number;
+    Colors color;
+    public ball(int number, Colors color){

```

Comment:
The constructor name `ball` should be capitalized as `Ball` to match the class name once it is corrected.